src/frequenz/sdk/config/_util.py

```diff
@@ -22,13 +22,22 @@ def load_config(
 
     The configuration class is expected to be a [`dataclasses.dataclass`][], which is
     used to create a [`marshmallow.Schema`][] schema to validate the configuration
-    dictionary.
+    dictionary using [`marshmallow.Schema.load`][] (which in turn uses the
+    [`marshmallow.Schema.load`][] method to do the validation and deserialization).
```

Comment:
`using [`marshmallow.Schema.load`][] (which in turn uses the [`marshmallow.Schema.load`]`

Wrong name?

Reply:
Nope: https://marshmallow.readthedocs.io/en/stable/marshmallow.schema.html#marshmallow.schema.Schema.load

(this would have been caught by the docs tests if it were wrong)

Reply:
Yes, the first one should be `class_schema`. Will update.